### PR TITLE
Minimize warnings in tests; Update requirements

### PR DESCRIPTION
**Pull request by scvannost**

### CODE REVIEW

File: functions/visualizations.py [hunk 1]
```diff
@@ -3,8 +3,15 @@
...
+# clustergrammer2 via sklearn uses np.float which is deprecated as of numpy==1.20
+import warnings
+
+warnings.filterwarnings(
+    action="ignore", category=DeprecationWarning, module="scikit-learn"
+)
 import pandas as pd
-from clustergrammer import Network as CGNetwork
+
+from clustergrammer2 import Network as CGNetwork
```
Comment (jacoblurye) [inline functions/visualizations.py:14]: Have you confirmed that this works with the current frontend code? We don't use clustergrammer2 on the frontend.

Comment (scvannost) [reply functions/visualizations.py:14]: not yet oops

Comment (scvannost) [reply functions/visualizations.py:14]: from comparing [CG2's initialization](https://github.com/ismms-himc/clustergrammer2/blob/d9f28fd8fbc861ad0833420f350c59a21d3f012e/clustergrammer2/clustergrammer_fun/initialize_net.py#L73) to [CG's](https://github.com/MaayanLab/clustergrammer-py/blob/ea4c517e860e4afd1063164d98192b87b845fa39/clustergrammer/initialize_net.py#L49), `net.viz` has the structure

Comment (scvannost) [reply functions/visualizations.py:14]: Image Pasted at 2021-5-4 17-44
a quick test shows that the results match the shape of that already in use on prod

**Review by jacoblurye — APPROVED** [review]

**Pull request merged**